src/XRP/xrp-error.ts

```diff
@@ -30,6 +31,14 @@ export default class XrpError extends Error {
     'Could not convert payment transaction: (transaction). Please file a bug at https://github.com/xpring-eng/Xpring-JS/issues',
   )
 
+  /**
+   * Encountered a protocol buffer formatted in contradiction to the logic of the XRPL.
+   */
+  static malformedProtobuf = new XrpError(
+    XrpErrorType.MalformedProtobuf,
+    'Encountered a protocol buffer formatted in contradiction to the XRPL business logic',
```

Comment:
perhaps "Encountered a protocol buffer with unexpected format" or something similar? i feel this error message could be simpler